### PR TITLE
port some release/1.81 changes to main

### PR DESCRIPTION
**Pull request by tshepang**
this is a port of https://github.com/ferrocene/ferrocene/pull/988

### CODE REVIEW

File: ferrocene/doc/release-notes/src/24.11.0.rst [hunk 1]
```diff
@@ -1,8 +1,6 @@
 .. SPDX-License-Identifier: MIT OR Apache-2.0
    SPDX-FileCopyrightText: The Ferrocene Developers
 
-:upcoming-release:
```
Comment (Hoverbear) [inline ferrocene/doc/release-notes/src/24.11.0.rst:4]: Note we can't remove this before release.

**Review by Hoverbear — CHANGES_REQUESTED** [review]
bors merge

File: ferrocene/doc/release-notes/src/24.11.0.rst (hunk 1 above)
Comment (tshepang) [reply ferrocene/doc/release-notes/src/24.11.0.rst:4]: will leave this open until release is done

**Review by Hoverbear — APPROVED** [review]
bors merge

**Pull request merged**